Refunds API request classes (#5967)
Automattic/woocommerce-payments: includes/core/server/request/class-list-charge-refunds.md

> # `List_Charge_Refunds` request class

Reviewer: Update document regarding the change in using `charge` parameter. I think you would review this doc and another doc for `Refund_Charge` too.

Author: Fixed in https://github.com/Automattic/woocommerce-payments/pull/5967/commits/898efe2d0a8cc4136c189aa4a48b63f057ab4c6d

[ℹ️ This document is a part of __WooCommerce Payments Server Requests__](../requests.md)

## Description

The `WCPay\Core\Server\Request\List_Charge_Refunds` class is used to construct the request for refunding a refunds for specific charge.

## Parameters

> When creating `List_Charge_Refunds` requests, the item ID must be provided to the `::create()` method. The identifier should be in the `ch_XXX` format.

Reviewer: Nitpick: 

```suggestion
When creating `List_Charge_Refunds` requests, the item ID must be provided to the `::create()` method. The identifier should be in the `ch_XXX` or `py_YYY` format.
```

# `List_Charge_Refunds` request class

[ℹ️ This document is a part of __WooCommerce Payments Server Requests__](../requests.md)

## Description

> The `WCPay\Core\Server\Request\List_Charge_Refunds` class is used to construct the request for refunding a refunds for specific charge.

Reviewer: ```suggestion
The `WCPay\Core\Server\Request\List_Charge_Refunds` class is used to construct the request for listing refunds of a specific charge.
```

## Parameters

| Parameter | Setter                         | Immutable | Required | Default value |
|-----------|--------------------------------|:---------:|:--------:|:-------------:|

> | `charge`  | `set_charge( string $charge )` |    Yes    |   Yes    |     None      |

Reviewer: ```suggestion
| `charge`  | `set_charge( string $charge_id )` |    Yes    |   Yes    |     None      |
```
